src/main/java/com/adobe/cq/commerce/graphql/client/impl/GraphqlClientConfiguration.java

```diff
@@ -23,11 +23,15 @@
 @ObjectClassDefinition(name = "CIF GraphQL Client Configuration Factory")
 public @interface GraphqlClientConfiguration {
 
-    public String CQ_GRAPHQL_CLIENT = "cq:graphqlClient";
-    public String DEFAULT_IDENTIFIER = "default";
+    String CQ_GRAPHQL_CLIENT = "cq:graphqlClient";
+    String DEFAULT_IDENTIFIER = "default";
 
-    public static final int MAX_HTTP_CONNECTIONS_DEFAULT = 20;
-    public static final boolean ACCEPT_SELF_SIGNED_CERTIFICATES = false;
+    int MAX_HTTP_CONNECTIONS_DEFAULT = 20;
+    boolean ACCEPT_SELF_SIGNED_CERTIFICATES = false;
+
+    int DEFAULT_CONN_TIMEOUT = 30000;
```

Comment:
I would change this to `5000` (5 seconds), 30 seconds is by far too large.

Reply:
Agree and done